backend/backend/events.py

```diff
@@ -53,6 +53,7 @@ def start_sse_stream(output_stream=sys.stdout):
     new_balances = {}
     turn = None
     turn_order = {}
+    test = True
```

Comment:
Is this still meant to be here? If it is, could the name be something more explanatory?

Reply:
I can't think of a more elegant way of doing it at the moment. I'll change the name though definitely thanks.